### PR TITLE
Marriage testcase 7

### PR DESCRIPTION
**Pull request by Eezi**

### CODE REVIEW

File: e2e/testcases/marriage/7-validate-witness-1-page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { createPIN, login, goToSection } from '../../helpers'
+
+test.describe('7. Validate Witness 1 details page', () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page, 'k.mweene', 'test')
+    await createPIN(page)
+    await page.click('#header_new_event')
+    await page.getByText('Marriage', { exact: true }).click()
+    goToSection(page, 'witnessOne')
+  })
```
Comment (jamil314) [inline e2e/testcases/marriage/7-validate-witness-1-page.spec.ts:11]:
```suggestion
    await goToSection(page, 'witnessOne')
```

**Review by jamil314 — CHANGES_REQUESTED** [review]

File: e2e/testcases/marriage/7-validate-witness-1-page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import { expect, test } from '@playwright/test'
+import { createPIN, login, goToSection } from '../../helpers'
+
+test.describe('7. Validate Witness 1 details page', () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page, 'k.mweene', 'test')
+    await createPIN(page)
+    await page.click('#header_new_event')
+    await page.getByText('Marriage', { exact: true }).click()
+    goToSection(page, 'witnessOne')
```
Comment (jamil314) [inline e2e/testcases/marriage/7-validate-witness-1-page.spec.ts]:
```suggestion
    await goToSection(page, 'witnessOne')
```

**Review by jamil314 — APPROVED** [review]

File: e2e/testcases/marriage/7-validate-witness-1-page.spec.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import { createPIN, login, goToSection } from '../../helpers'
+
+test.describe('7. Validate Witness 1 details page', () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page, 'k.mweene', 'test')
+    await createPIN(page)
+    await page.click('#header_new_event')
+    await page.getByText('Marriage', { exact: true }).click()
+    await goToSection(page, 'witnessOne')
+
```
Comment (jamil314) [inline e2e/testcases/marriage/7-validate-witness-1-page.spec.ts:12]: `test.beforeEach` should end here

**Review by jamil314 — CHANGES_REQUESTED** [review]

File: e2e/testcases/marriage/7-validate-witness-1-page.spec.ts (hunk 3 above)
Comment (Eezi) [reply e2e/testcases/marriage/7-validate-witness-1-page.spec.ts:12]: Glad you noticed this!👌🏼

**Review by jamil314 — APPROVED** [review]

**Pull request merged**